airsonic-main/src/main/java/org/airsonic/player/controller/SubsonicRESTController.java

```diff
@@ -164,13 +165,13 @@ public void getLicense(HttpServletRequest request, HttpServletResponse response)
         request = wrapRequest(request);
         License license = new License();
 
-
         license.setEmail("airsonic@github.com");
         license.setValid(true);
-        Date farFuture = new Date();
-        farFuture.setYear(farFuture.getYear() + 100);
-        license.setLicenseExpires(jaxbWriter.convertDate(farFuture));
-        license.setTrialExpires(jaxbWriter.convertDate(farFuture));
+        Calendar calendar = Calendar.getInstance();
+        calendar.set(Calendar.YEAR, calendar.get(Calendar.YEAR) + 1900 + 100);
+        XMLGregorianCalendar georgianCalendar = jaxbWriter.convertCalendar(calendar);
```

Comment:
While I think you meant this to be `gregorianCalendar`, I would prefer if you kept the name as something more meaningful, like the original `farFuture`, or maybe `farFutureCalendarDate`.

And I think you could avoid having the `calendar` variable as well as simplifying the date calculation by chaining the method calls:
`XMLGregorianCalendar farFuture = jaxbWriter.convertCalendar(Calendar.getInstance().add(Calendar.YEAR, 100));`

Reply:
Unfortunately, the `.add` method returns `void` :/